cfme/tests/configure/test_paginator.py

```diff
@@ -88,6 +89,8 @@ def schedule(appliance):
     schedule.delete()
 
 
+@pytest.mark.meta(blockers=[
```

Comment:
Q: as far as I understand this BZ blocks only one or two test cases not whole test, right ?
Please update statement if so.

Reply:
Thanks for the review, you're right, BZ affects not the whole test. Only those test cases for VMDB Summary view.
Updated the blocker statement.